test/parallel/test-inspector-invalid-args.js

```diff
@@ -1,11 +1,12 @@
 'use strict';
+const common = require('../common');
```

Comment:
@Trott this is not whitespace only. I can remove all the other whitespace only changes if you'd like, I think they are more readable, and I think they are implied by the test guide, but this change, specifically, brings this test into conformance with the guide.